CRM/Contact/Import/Form/Summary.php

```diff
@@ -128,7 +128,7 @@ public function postProcess() {
 
     $importTableName = $this->get('importTableName');
     // do a basic sanity check here
-    if (strpos($importTableName, 'civicrm_import_job_') === 0) {
+    if (strpos($importTableName, 'civicrm_tmp') === 0 || strpos($importTableName, 'civicrm_import_job') === 0) {
```

Comment:
Why do any of this? Why not just DROP TABLE IF EXISTS - also is this a temporary table? If so it should be DROP TEMPORARY TABLE IF EXISTS (respects transactions )

Reply:
Doh' good point these are actually not temporary tables doh *seamus needs more coffee*